Modules/Filtering/ImageGrid/test/itkExpandImageFilterTest.cxx

```diff
@@ -232,58 +233,27 @@ itkExpandImageFilterTest(int, char *[])
     ++streamIter;
   }
 
+  // Test error handling
 
-  if (!testPassed)
-  {
-    std::cout << "Test failed." << std::endl;
-    return EXIT_FAILURE;
-  }
+  std::cout << "Setting Input to nullptr" << std::endl;
+  expander->SetInput(nullptr);
 
-  // Test error handling
+  ITK_TRY_EXPECT_EXCEPTION(expander->Update());
 
-  try
-  {
-    testPassed = false;
-    std::cout << "Setting Input to nullptr" << std::endl;
-    expander->SetInput(nullptr);
-    expander->Update();
-  }
-  catch (const itk::ExceptionObject & err)
-  {
-    std::cout << err << std::endl;
-    expander->ResetPipeline();
-    expander->SetInput(input);
-    testPassed = true;
-  }
 
-  if (!testPassed)
-  {
-    std::cout << "Test failed." << std::endl;
-    return EXIT_FAILURE;
-  }
+  expander->ResetPipeline();
+  expander->SetInput(input);
 
+  std::cout << "Setting Interpolator to nullptr" << std::endl;
+  expander->SetInterpolator(nullptr);
 
-  try
-  {
-    testPassed = false;
-    std::cout << "Setting Interpolator to nullptr" << std::endl;
-    expander->SetInterpolator(nullptr);
-    expander->Update();
-  }
-  catch (const itk::ExceptionObject & err)
-  {
-    std::cout << err << std::endl;
-    expander->ResetPipeline();
-    expander->SetInterpolator(interpolator);
-    testPassed = true;
-  }
+  ITK_TRY_EXPECT_EXCEPTION(expander->Update());
+
+
+  expander->ResetPipeline();
+  expander->SetInterpolator(interpolator);
 
-  if (!testPassed)
-  {
-    std::cout << "Test failed." << std::endl;
-    return EXIT_FAILURE;
-  }
 
   std::cout << "Test passed." << std::endl;
-  return EXIT_SUCCESS;
+  return testPassed;
```

Comment:
wrong, _testPassed_ is type _bool_

Reply:
Had spotted that. Thanks for the heads-up. Fixed in a36b621.